test-files/ui/features/step_definitions/custom_steps.rb

```diff
@@ -476,7 +476,8 @@
 
 Then(/^I should see element "([^"]*)" with no value and placeholder "([^"]*)"$/) do |id, value|
   find(id).value.should eq ""
-  page.find(:css, 'input[placeholder="' + value + '"]')
+  #page.find(:css, 'input[placeholder="' + value + '"]')
+  find(id, 'input[placeholder="' + value + '"]')
```

Comment:
looking in the capybara api for how this works.
http://www.rubydoc.info/github/jnicklas/capybara/Capybara/Node/Finders#find-instance_method
Seems like find is taking two selector arguments?